### PR TITLE
fix: typos in shipping settings

### PR DESCRIPTION
**Pull request by rjchow**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Some mis-capitalisation of 'WooCommerce' was pointed out and fixed - the search was run across the whole woocommerce-admin but there were no instances outside of this.

Also fixed two typos.

### How to test the changes in this Pull Request:
Same as https://github.com/woocommerce/woocommerce/pull/33801 but without the spelling and capitalisation mistakes.

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce-admin/client/shipping/test/shipping-recommendations-wrapper.test.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ describe( 'ShippingRecommendations', () => {
 			/>
 		);
 
-		expect( queryByText( 'Woocommerce Shipping' ) ).not.toBeInTheDocument();
+		expect( queryByText( 'WoCcommerce Shipping' ) ).not.toBeInTheDocument();
```
Comment (ilyasfoo) [inline plugins/woocommerce-admin/client/shipping/test/shipping-recommendations-wrapper.test.tsx]: There seems to be a few of these in this file:

```suggestion
		expect( queryByText( 'WooCommerce Shipping' ) ).not.toBeInTheDocument();
```

Comment (rjchow) [reply plugins/woocommerce-admin/client/shipping/test/shipping-recommendations-wrapper.test.tsx]: I really do need a new keyboard and eyes 🥺

**Review by ilyasfoo — CHANGES_REQUESTED** [review]
It seems there's `shippping` here

https://github.com/woocommerce/woocommerce/blob/fix/typos-in-shipping-settings/plugins/woocommerce-admin/client/shipping/shipping-tour.tsx#L268

**Comment by rjchow** [comment]
> It seems there's `shippping` here
> 
> https://github.com/woocommerce/woocommerce/blob/fix/typos-in-shipping-settings/plugins/woocommerce-admin/client/shipping/shipping-tour.tsx#L268

I actually have a new keyboard on order

**Review by ilyasfoo — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**